### ISSUE TITLE
HACS showing updates available over already updated elements. It repeats after a reboot

### ISSUE THREAD

**Issue by molly200** [hacs/integration#2728]
### Web browser

Chrome

### Web browser version

Version 1.5.115 Chromium: 80.0.3987.149 (Official Build) (64-bit)

### System Health details

<html>
<body>

Version | core-2022.6.0
-- | --
Installation Type | Home Assistant Supervised
Development | false
Supervisor | true
Docker | true
User | root
Virtual Environment | false
Python Version | 3.9.12
Operating System Family | Linux
Operating System Version | 5.10.0-14-amd64
CPU Architecture | x86_64
Timezone | America/Vancouver

</body>
</html>

### Checklist

- [X] I'm running the newest version of HACS <https://github.com/hacs/integration/releases/latest>
- [X] I have filled out the issue template to the best of my ability.
- [X] I have read <https://hacs.xyz/docs/issues>
- [X] This issue is related to the frontend of HACS.
- [X] This issue only contains 1 issue (if you have multiple issues, open one issue for each issue).
- [X] This issue is not a duplicate issue of currently [open](https://github.com/hacs/integration/issues) or issues [pending release](https://github.com/hacs/integration/issues?q=is%3Aissue+is%3Aclosed+sort%3Aupdated-desc+milestone%3Anext).

### Describe the issue

updated Hacs from version 1.24.1 to 1.25.5 latest. After reboot my cards are not loaded in lovelace. HACS finds what appears to be the hexadecimal version number of the numeric version number as an update. All the elements are already updated to the latest version.  after a few minutes the number of cards settle to unspecific not loaded cards. A reboot  will restart the process from the top. 

 

### Reproduction steps

1. Don't want to duplicate, as it appears that the issue was reported twice in the last 10 days
2. Reverting back to version 1.24.1 resolves the issue
3.
...

### Screenshots

![HACS_1](https://user-images.githubusercontent.com/65462223/171761234-6c0ca77d-0046-44d1-95ef-8a1d8eea3268.PNG)
![HACS_2](https://user-images.githubusercontent.com/65462223/171761235-8cd72cfb-09ef-4076-aa72-b0aea3d1965b.PNG)

### Javascript logs from your browser console

```text
I am not a computer savvy. I don't know what this refers to
```

### Diagnostics dump

I am not a computer savvy. I don't know what this refers to

**Comment by ludeeus** [comment]
Please open an issue with the integration/backend template, there is nothing here to go on 👍

**Comment by molly200** [comment]
Sorry, I don't speak your language. I'm a simple HA user. Received an hacs update notification and updated. Did not work and opened an issue with the source. 
 
What  does "integration/backend template, there is nothing here to go on" mean? 

As a simple user, I would imagine an update will make things better, not worst! Happy to go back to the working version, until it breaks again and someone understanding this language can help fix the problem. 

Did not want to be ignorant, and did read the past posts on the issue. These posts received no answer from you and have been closed in the same manner. I assumed the information provided and the changes from a working version to one that's not working, will be enough to track down the problem. 

Aside from the time I've already invested in having HA installed and working, along with all the frustrations in having to maintain a working version,  I have no interest in spending hours and hours of reading and learning. I just want to use HA and have it working. I wonder if that will ever be possible! 

Thank you for your work and have a nice day!

**Comment by ludeeus** [comment]
Its the first item in the list
https://github.com/hacs/integration/issues/new/choose

**Comment by molly200** [comment]
Man, we must be living on different planets!

Wow! I don't want to be mean and judgmental, but with the short reply I'm afraid that you can't see past or any further to whatever it bothers you. The way I feel right now, is that you want one to follow your rules on something that you engage people into, using your project and then help you fix whatever does not work, with the expectation of following unreasonable rules.  

I don't understand the integration's frontend vs. backend and I did complete the form to the best of my abilities.  With frustration, I did what you wanted, but I don't give a $#!t about your rules. If you want people to get engaged in you project, the least you can do, is to be thankful for getting feedback. You could have moved this template in the right place yourself, or better yet could have addressed it immediately, but you fail to listen and give respect to other people's wishes.

In conclusion, in my view you can do what you want with your project, as I/we have no control over it. Telling people what to do, is a different story. Don't fix it, if is to much trouble for you! 

Have a good one!
